src/main/kotlin/org/web3j/container/IKGenericContainer.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2019 Web3 Labs Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.web3j.container
+
+import org.web3j.protocol.Web3jService
+
+interface IKGenericContainer {
+    fun startNode()
+
+    fun stop()
+
+    fun createService(): Web3jService
```

Comment:
Can the `start()` method return a `Web3jService`? - then the `ContainerService` will start a container and create a web3j with the injected url and `LocalService` will just return your localweb3jservice thingy in web3j-evm?

Reply:
I'll rename startNode to startService (see comment above), and then have this return the service, removing createService.